app/views/standard/additionalMetadataNavigation.scala.html

```diff
@@ -53,6 +53,22 @@ <h2 class="govuk-error-summary__title">There is a problem</h2>
     </div>
   }
 }
+@displayFileStatus(file: File, metadataType: String) = {
```

Comment:
Is it possible to move this logic out of the template and into the controller? Its better to avoid putting too much logic into the templates. Maybe change the `File` case class to something like this:

```
case class StatusTag(text: String, colour: String)

case class File(id: UUID, name: String, fileType: Option[String], children: List[File], statusTag: StatusTag)
```
There needs to be something similar for the view transfers page, so it would be good to create a common object `StatusTag` that can be used for different tagging scenarios.

Reply:
Yeah, good point, thanks.